src/actions/index.js

```diff
@@ -0,0 +1,12 @@
+import { CREATE_BOOK,REMOVE_BOOK } from "../constaints/action-types";
+
+export const createBook = book => ({
+        type : CREATE_BOOK,
+        payload: book,
```

Comment:
[Optional]
- Kindly format the code nicely, it looks like this has about 8 spaces instead of the 4 spaces